### PR TITLE
Change Control subtype to UserControl

### PR DESCRIPTION
**Pull request by davidwengier**
Fix the enum value for user controls.

### CODE REVIEW

File: src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Rules/Items/AdditionalFiles.BrowseObject.xaml [hunk 1]
```diff
@@ -140,7 +140,7 @@
     </EnumProperty.DataSource>
     <EnumValue Name="Designer" />
     <EnumValue Name="Component" />
-    <EnumValue Name="Control" />
+    <EnumValue Name="UserControl" />
     <EnumValue Name="Form" />
```
Comment (davkean) [inline src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Rules/Items/AdditionalFiles.BrowseObject.xaml]: This is also wrong, this doesn't exist.

**Comment by davkean** [comment]
Is this the underlying cause of https://github.com/dotnet/roslyn/issues/36609?

File: src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Rules/Items/AdditionalFiles.BrowseObject.xaml (hunk 1 above)
Comment (davidwengier) [reply src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Rules/Items/AdditionalFiles.BrowseObject.xaml]: `Form`? Okay, I shall remove.

**Comment by davidwengier** [comment]
Yes and no. @lifengl identified three issues in an email, and this is one of them, however he's also changed CPS so the values here don't matter any more, if I understand correctly.

File: src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Rules/Items/AdditionalFiles.BrowseObject.xaml (hunk 1 above)
Comment (davidwengier) [reply src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Rules/Items/AdditionalFiles.BrowseObject.xaml]: Where is the official list of subtypes? `Form` is used in [our own code](https://github.com/dotnet/project-system/blob/afa02d302e08df1ffe8689dbcd6b9d65c6f59c2f/src/Microsoft.VisualStudio.AppDesigner/Microsoft.VisualStudio.AppDesigner.vbproj#L50) in a couple of spots, and is part of legacy: http://index/?query=%60VSSUBTYPE_FORM&rightProject=VS(Repo)&file=src%5Cvsproject%5Cinc%5Csubtypes.h&line=25

Comment (davkean) [reply src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Rules/Items/AdditionalFiles.BrowseObject.xaml]: Yeah my bad, I even handled this in https://github.com/dotnet/project-system/pull/4913. I was originally under the impression that Form/Designer were the same, but they aren't: http://index/?query=Form&rightProject=System.Windows.Forms&file=System%5CWinForms%5CForm.cs&line=49. Let me find which control uses "Designer".

**Pull request merged**